### PR TITLE
Add mastodon bridge

### PR DESCRIPTION
**Pull request by karloluiten**
Add the mastodon bridge to twitter

### CODE REVIEW

File: Using-Mastodon/FAQ.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ We are using the OStatus suite of protocols:
 
 #### What is mastodon.social?
 
-The "flagship" instance of Mastodon, aka the server I run myself with the latest code. It's not supposed to be the only instance in the end.
+The "flagship" instance of Mastodon, aka the server I run myself with the latest code. There are more mastodon instances. There is an updated list at [instances.mastodon.xyz](https://instances.mastodon.xyz).
```
Comment (wxcafe) [inline Using-Mastodon/FAQ.md]: Please make a separate PR for this

**Comment by dariusk** [comment]
Closing, as the `Apps.md` change has since been added to the main documentation.
